### DISCUSSION TITLE
Posibility to open a local instance to the network

### DISCUSSION THREAD

**Discussion by StellarNear** [open-webui/open-webui#1143] [General]
First of all huge thanks for your amazing project ! 

Would it be possible to have a sort of "--listen" as it's done in Automatic1111 to make the open-webui instant that is running locally also available from a local network connection. Maybe as an option/calling argument.

Use case : 
I have a computer hosting the open-webui instance, i want to browser the url from my phone (connected to the same network in WIFI) and chat with the instance from it.

**Comment by tjbck** [comment]
If you're using docker, it should be open by default!

**Reply by StellarNear** [reply]
i'm not (i'm running it on the Pinokio hub)

**Reply by Xerophayze** [reply]
I would like to know the same thing.  Same setup.  running it through Pinokio Hub.

**Reply by alpha754293** [reply]
@tjbck 
I am using Open WebUI via a Docker container via this command:

`sudo docker run -d -p 3000:8080 --gpus all --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:cuda`

Which is running in an Ubuntu 22.04 WSL2 instance that's running on my Windows 10 system.

The IP address that it picks up is internal to the docker network and isn't accessible from outside of said Docker network.

**Reply by tcarwash** [reply]
let me preface this with a "I don't know anything about WSL" but,

maybe try `-p 0.0.0.0:3000:8080` 

by default the container only exposes the service on it's docker network interface which would probably only be accessible from the WSL host. specifying 0.0.0.0 (listen on all IPs) should listen on whatever WSL's IP is on your system, which should i think be accessible from windows 10 on your system (I'm not sure if it will be on localhost:3000 or some other IP or hostname). 

hopefully if that doesn't fix it, it at least gives you some place to start looking!
